docs/content/en/docs/observability/prometheus.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Metrics with Prometheus"
+weight: 10
+---
+
+
+# Metrics in Fission
+
+Fission exposes metrics in the Prometheus standard, which can be readily scraped and used using a Prometheus server and visualized using Grafana. The metrics help monitor the state of the Functions as well as the Fission components.
+
+
+### Prometheus
+
+Prometheus is a monitoring and alerting tool. It uses a multi-dimensional data model with time series data identified by metric name and key/value pairs. 
+
+Fission exposes metrics which are pulled and operated by Prometheus at regular intervals.
+
+### Grafana
+
+Grafana is a visualizing tool which can query, visualize, alert on and understand metrics. It supports Prometheus as it's data source.
+
+
+# Setting up
+
+There are different ways to install Prometheus. It can be installed and run in and outside containers. Since Fission itself runs in Kubernetes, we'll use the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) which is a way of installing Prometheus as Kubernetes Custom Resource.
+
+## Prerequisite
+
+- Kubernetes cluster
+- Fission [installed in the cluster](https://docs.fission.io/docs/installation/)
+- [Helm](https://helm.sh/) (This post assumes helm 3 in use)
+- kubectl and kubeconfig configured
+
+
+## Install Prometheus and Grafana
+
+We'll install Prometheus and Grafana in a namespace named `monitoring`.
+
+To create the namespace, run the following command in a terminal:
+
+```
+$ expose METRICS_NAMESPACE=monitoring
+$ kubectl create namespace $METRICS_NAMESPACE
+```
+
+Install Prometheus and Grafana with the release name `fission-metrics`.
+
+```
+$ helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+$ helm repo update
+$ helm install fission-metrics --namespace $METRICS_NAMESPACE prometheus-community/kube-prometheus-stack --set kubelet.serviceMonitor.https=true --set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false --set prometheus.prometheusSpec.podMonitorSelectorNilUsesHelmValues=false --set prometheus.prometheusSpec.ruleSelectorNilUsesHelmValues=false
```

Comment:
Can we make this command multiline readable by adding \ so that it is easier to read!